### PR TITLE
stall-analyser: gracefully handle empty input

### PR DESCRIPTION
**Pull request by bhalevy**
Currently, the tools prints an obscure error
if the input is empty:
```
Traceback (most recent call last):
  File "/home/bhalevy/dev/scylla_s3_reloc_server/./seastar//scripts/stall-analyser.py", line 363, in <module>
    print_stats(tally, tmin)
                       ^^^^
NameError: name 'tmin' is not defined. Did you mean: 'min'?
```

Instead, print a meaningful error and direct the user to run stall-analyser.py --help for usage instructions.

Ref https://github.com/scylladb/scylla_s3_reloc_server/pull/103

### CODE REVIEW

File: scripts/stall-analyser.py [hunk 1]
```diff
@@ -368,11 +375,13 @@ def main():
...
         if address_threshold:
             trace = list(dropwhile(lambda addr: int(addr, 0) >= address_threshold, trace))
-        tmin = args.minimum or 0
         if t >= tmin:
             graph.process_trace(trace, t)
 
     try:
+        if not graph:
+            print("No input data found. Please run `stall-analyser.py --help` for usage instruction")
+            exit(0)
```
Comment (tchaikov) [inline scripts/stall-analyser.py:384]: i'd suggest use `sys.exit()` instead. see https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module . i am quoting the remarks in the document.

> They are useful for the interactive interpreter shell and should not be used in programs.

Comment (bhalevy) [reply scripts/stall-analyser.py]: done

**Review by tchaikov — APPROVED** [review]
lgtm

**Comment by bhalevy** [comment]
@scylladb/seastar-maint please merge

**Pull request closed without merging**